src/client/pages/OfferNew/FaqSection.tsx

```diff
@@ -151,28 +151,60 @@ export const Accordion: React.FC<AccordionProps> = ({ headline, body }) => {
   )
 }
 
-export const FaqSection: React.FC = () => {
-  const pathLocale = useCurrentLocale()
-  const language = getIsoLocale(pathLocale)
-  const faqs = useFaqsQuery({ variables: { language } })
-  const languageData = faqs?.data?.languages[0]
+type FaqPanelProps = {
+  quote: BundledQuote
+}
+
+const FaqPanel = ({ quote }: FaqPanelProps) => {
+  const { isoLocale } = useCurrentLocale()
+  const { data: faqItems } = useContractFaqQuery({
```

Comment:
Hopefully we can make an endpoint in Giraffe that gets all FAQ content at once, so we can call it in `FaqSection` instead

Reply:
That would be nice!